<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>huaban_Form</class>
 <widget class="QWidget" name="huaban_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1800</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1800</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1800</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>同步课堂</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Images/images.qrc">
    <normaloff>:/new/prefix1/Images/logo.png</normaloff>:/new/prefix1/Images/logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame{
	background-color: rgb(240, 240, 240);
 border-style: solid;
	border-color: rgb(240, 240, 240);
 border-width: 2px;
    border-radius: 30px;
}
QWidget#page_1,#page_2,#page_3,#page_4,#page_5{
	background-color: rgb(240, 240, 240);
}
QWidget#widget,#page{
	background-color: rgb(255, 255, 255);
}
QLabel{
	background-color: rgb(240, 240, 240);
	color: rgb(85, 170, 255);
	font: 20pt &quot;微软雅黑&quot;;
}
QPushButton {
    border-style: solid;
    border-width: 2px;
    border-color:rgba(200, 200, 200,0); 
    border-radius: 30px;

    background-color:transparent;
    background-repeat: no-repeat;
    background-position: left center;
}
QPushButton:hover{
	color: red;
	border-color:rgba(200,200,200,0.1);
    background-color:rgba(200,200,200,0.1);
}
QPushButton:pressed{
	color:rgba(0,255,0);
	border-color:rgba(200,200,200,1);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>1600</width>
       <height>900</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1800</width>
       <height>900</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1" colspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>739</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>/</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="4" colspan="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>739</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="7">
       <widget class="QPushButton" name="pushButton_huabanclose">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Images/images.qrc">
          <normaloff>:/new/prefix1/Images/Flase.png</normaloff>:/new/prefix1/Images/Flase.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>190</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1" rowspan="3" colspan="6">
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="minimumSize">
         <size>
          <width>1600</width>
          <height>500</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255,1);</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="drawwidget" name="page_1"/>
        <widget class="drawwidget" name="page_2"/>
        <widget class="drawwidget" name="page_3"/>
        <widget class="drawwidget" name="page_4"/>
        <widget class="drawwidget" name="page_5"/>
       </widget>
      </item>
      <item row="1" column="7">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>190</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton_huabanleft">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Images/images.qrc">
          <normaloff>:/new/prefix1/Images/指向-left.png</normaloff>:/new/prefix1/Images/指向-left.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="7">
       <widget class="QPushButton" name="pushButton_huabanright">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Images/images.qrc">
          <normaloff>:/new/prefix1/Images/指向-right.png</normaloff>:/new/prefix1/Images/指向-right.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>189</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="7">
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>190</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="1">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>317</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="2" rowspan="2" colspan="3">
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>1000</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1200</width>
          <height>150</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>画笔橡皮</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="QPushButton" name="huaban_huabi">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="Images/images.qrc">
                <normaloff>:/new/prefix1/Images/画笔.png</normaloff>:/new/prefix1/Images/画笔.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="huaban_era">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="Images/images.qrc">
                <normaloff>:/new/prefix1/Images/橡皮.png</normaloff>:/new/prefix1/Images/橡皮.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>画笔粗细</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QPushButton" name="pushButton_3">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="Images/images.qrc">
                <normaloff>:/new/prefix1/Images/点.png</normaloff>:/new/prefix1/Images/点.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="pushButton_4">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="Images/images.qrc">
                <normaloff>:/new/prefix1/Images/点 (1).png</normaloff>:/new/prefix1/Images/点 (1).png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="pushButton_5">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="Images/images.qrc">
                <normaloff>:/new/prefix1/Images/点 (2).png</normaloff>:/new/prefix1/Images/点 (2).png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="styleSheet">
            <string notr="true">QPushButton {
    border-style: solid;
    border-width: 2px;
    border-color:rgb(200, 200, 200); 
    border-radius: 5px;
}
QPushButton:hover{
	color: red;
	border-color:rgba(200,200,200,0.5);
    background-color:rgba(200,200,200,0.5);
}
QPushButton:pressed{
	color:rgba(0,255,0);
	border-color:rgba(200,200,200,1);
}</string>
           </property>
           <property name="title">
            <string>画笔颜色</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QPushButton" name="pushButton_heise">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(0, 0, 0);</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="pushButton_chengse">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:rgb(255, 170, 0);</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="pushButton_huangse">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 0);</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QPushButton" name="pushButton_huise">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(100,100,100);</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QPushButton" name="pushButton_hongse">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 0, 0);</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="pushButton_baise">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pushButton_lvse">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(0,255,0);</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="pushButton_lanse">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:rgb(50, 100, 150);
</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QPushButton" name="pushButton_qingse">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(0,255,255);</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="1" column="4">
             <widget class="QPushButton" name="pushButton_fense">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:rgb(255, 85, 255);</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>撤销删除</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QPushButton" name="huaban_unm">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="Images/images.qrc">
                <normaloff>:/new/prefix1/Images/27撤销.png</normaloff>:/new/prefix1/Images/27撤销.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>35</width>
                <height>35</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="huaban_del">
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="Images/images.qrc">
                <normaloff>:/new/prefix1/Images/删 除.png</normaloff>:/new/prefix1/Images/删 除.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>35</width>
                <height>35</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="5">
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>317</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="5" column="6" colspan="2">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>71</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>drawwidget</class>
   <extends>QWidget</extends>
   <header location="global">drawwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Images/images.qrc"/>
 </resources>
 <connections/>
</ui>
